web/src/main/angular/src/app/core/components/transaction-list-bottom-contents/index.ts

```diff
@@ -12,6 +12,7 @@ import { SyntaxHighlightPopupModule } from 'app/core/components/syntax-highlight
 import { TransactionListBottomContentsContainerComponent } from './transaction-list-bottom-contents-container.component';
 import { HelpViewerPopupModule } from 'app/core/components/help-viewer-popup';
 import { ServerErrorPopupModule } from 'app/core/components/server-error-popup';
+import {TransactionTimelineModuleV2} from "../transaction-timeline-v2";
```

Comment:
Use absolute path and single quotes for the consistency.

```suggestion
import { TransactionTimelineModuleV2 } from 'app/core/components/transaction-timeline-v2';
```